config/filter.d/lighttpd-auth.conf

```diff
@@ -4,7 +4,13 @@
 [Definition]
 
 failregex = ^: \(http_auth\.c\.\d+\) (password doesn\'t match .* username: .*|digest: auth failed for .*: wrong password|get_password failed), IP: <HOST>\s*$
+            ^: \(\S+\.c\.\d+\) user \S* ?(not found|not authorized|password mismatch|sent invalid token|misbehaved): .+, IP: <HOST>
```

Comment:
We should then advise web-app developers not to put (unverified) data from clients into their failure message, which should simply contain for example the name of their application.
We could use `\S+` but I think it is too restrictive.